.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: elixir
+elixir:
+  - 1.3.4
+otp_release:
+  - 19.1
+
+script:
+  - MIX_ENV=test mix do compile --warnings-as-errors, coveralls.json
```

Comment:
`MIX_ENV=test mix do compile --warnings-as-errors` is very strict!

You can also run `mix coveralls.travis`: https://github.com/parroty/excoveralls#mix-coverallstravis-post-coverage-from-travis